libpod/pod_api.go

```diff
@@ -432,7 +432,7 @@ func (p *Pod) Inspect() (*PodInspect, error) {
 	var (
 		podContainers []PodContainerInfo
 	)
-
+	status, _ := p.GetPodStatus()
```

Comment:
This should happen after we update the pod, so move it to line 441.
And check for the error that is returned from this call, don't suppress it

Reply:
fixed.